### ISSUE TITLE
Wrong file name in modsec-sdbm-util.c (compile instruction) 

### ISSUE THREAD

**Issue by jschleus** [SpiderLabs/modsec-sdbm-util#4]
At the top of the file modsec-sdbm-util.c there is the very helpful comment:

```
 /*
  * Use the follwoing command to get it compiled: 
  * gcc modsec-sdbm.c -g `pkg-config apr-1 apr-util-1 --libs --cflags` -o modsec-sdbm
  */
```

but the file seems renamed so that the comment should be adapted (small typo also corrected): 

```
  /*
  * Use the following command to get it compiled: 
  * gcc modsec-sdbm-util.c -g `pkg-config apr-1 apr-util-1 --libs --cflags` -o modsec-sdbm-util
  */    
```

By the way it would be nice to have some short build informations additionally for e.g. in the README.md file since I first unsuccessfully spent some time how to use the also available autogen.sh and Makefile.am files).

**Comment by bostrt** [comment]
@jschleus Hopefully my pull request (https://github.com/SpiderLabs/modsec-sdbm-util/pull/7) will get accepted. It adds some build steps to README.md.

**Comment by zimmerle** [comment]
Hi @jschleus,

This issue was closed by @bostrt's PR at #9.

Thanks!
